Find a functionally equivalent code:android.media.MediaRouter.removeRouteInt:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * @hide internal use only
 */

@@ -1,3 +1,3 @@
 {
-    removeRoute(info);
+    removeRouteStatic(info);
 }

